test/unit/blocks_looks.js

```diff
@@ -23,6 +25,146 @@ const fakeRuntime = {
 };
 const blocks = new Looks(fakeRuntime);
 
+test('switch costume block runs correctly', t => {
+    /**
+     * Test which costume index the `switch costume`
+     * block will jump to given an argument and array
+     * of costume names.
+     *
+     * @param {string[]} costumes List of costume names as strings
+     * @param {string|number|boolean} arg The argument to provide to the block.
+     * @param {number} [currentCostume=1] The 1-indexed default costume for the sprite to start at.
+     * @return {number} The 1-indexed costume index on which the sprite lands.
+     */
+    const testCostume = (costumes, arg, currentCostume = 1) => {
+        const rt = new Runtime();
+        const looks = new Looks(rt);
+
+        const sprite = new Sprite(null, rt);
+        const target = new RenderedTarget(sprite, rt);
+
+        sprite.costumes = costumes.map(name => ({name: name}));
+        target.currentCostume = currentCostume - 1; // Convert to 0-indexed.
+
+        looks.switchCostume({COSTUME: arg}, {target});
+
+        return target.currentCostume + 1; // Convert to 1-indexed.
+    };
+
+    // Non-existant costumes do nothing
+    t.strictEqual(testCostume(['a', 'b', 'c', 'd'], 'e', 3), 3);
+
+    // Difference between string and numeric arguments
```

Comment:
These tests are awesome and very thorough!

Could you clarify this comment a little bit e.g. "Number arguments are always treated as the costume index. String arguments are treated as the name of the costume, otherwise converted to the costume index if possible"  and add one more test case to illustrate all the possible cases here
`t.strictEqual(testCostume(['a', 'b', 'c'], '2'), 2);`